### DISCUSSION TITLE
Time of "sensor expired" is only for English speakers ok

### DISCUSSION THREAD

**Discussion by Lutz9357** [NightscoutFoundation/xDrip#3615] [General]
Please add a new function to Xdrip+. 
I haven't the possibility to change the time format In "sensor expired at" from PM and AM into 24 hour format.
I read now (in german) "Sensor endet um 8.30 PM" - In future please "Sensor endet um 20.30 Uhr"
The english time format is confusing for me.

**Comment by Navid200** [comment]
Thanks for the suggestion.  
This has come up before.  
I'm not sure if this is possible or not.  But, for now, assuming it is, would you please click on Create issue from discussion in the right pane so that this is converted to an issue?  

I will see if I can do this easily.  If it is not possible or if there has been a reason for intentionally leaving this out, we may have to close the issue.  

Thanks

**Reply by Lutz9357** [reply]
Thank you

**Comment by PhilTU** [comment]
It isn't an "English" time, but the choice of 12 / 24h format on the phone clock could perhaps be respected by xDrip ?

**Reply by Lutz9357** [reply]
The clock of my phone has the 24h format.

**Reply by Navid200** [reply]
Thanks for submitting the request.
It is my plan to implement this.  Unless a developer tells me that this is the way it is for a reason, you will get your request. 
Please be patient now.

**Reply by Lutz9357** [reply]
@Navid200 Thank you very much

**Reply by TWilhoWI** [reply]
Any progress on this?

**Reply by Navid200** [reply]
@TWilhoWI 
No,
We have open issues that are 6 years old.  
We don't have many developers.
We have lots of open PRs.  Even if a fix is submitted, it will take a long time to review it.

This is because there are not many developers working on xDrip.  We are all volunteers.  
Do you want to help?  Do you know Java?  Do you have time?
If your answer is no to either or both, you have to be patient.
